homework-submission/todo-app/src/components/todo.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './todo.css';
+const Todo = props => {
+  const { description, id, handleDone, done, deadline, onDelete } = props;
+  return (
+    <li className={done ? 'isDone' : ''}>
+      <span onClick={() => handleDone(id)}>
```

Comment:
Nice work! You want straight for the homework of week 2, implementing click events! :-)